Task-WebSocket.md

```diff
@@ -0,0 +1,98 @@
+# SaltyRTC WebSocket Task
+
+This task uses the end-to-end encrypted WebSocket connection set up by
+the signalling channel to send user defined messages. Note that there is
+no handover to a separate signaling channel.
+
+# Conventions
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT",
+"SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this
+document are to be interpreted as described in
+[RFC 2119](https://tools.ietf.org/html/rfc2119).
+
+# Terminology
+
+All terms from the [SaltyRTC protocol
+specification](./Protocol.md#terminology) are valid in this document.
+
+# Task Protocol Name
+
+The following protocol name SHALL be used for task negotiation:
+
+`v0.websocket.tasks.saltyrtc.org`
+
+# Task Data
+
+This task does not currently make use of the *data* field of the 'auth'
+messages described in the [SaltyRTC protocol
+specification](./Protocol.md#auth-message). The task data sent by the
+peers should be `Nil`.
+
+# Message Structure
+
+The same message structure as defined in the [SaltyRTC protocol
+specification](./Protocol.md#message-structure) SHALL be used.
+
+For simplicity, even though the `source` and `destination` fields in the
+nonce are not necessary anymore, the none structure is left unchanged.
+
+## 'close' Message
+
+The message itself and the client's behaviour is described in the
```

Comment:
This section shouldn't be needed unless you want to define special behaviour (that's why the WebRTC task has it).

Reply:
It's good to be explicit.

Reply:
I disagree in this case because it really adds no value. Mentioning that 'close' is expected to work can just as well be in *Client-to-Client Messages*.

Reply:
I did that in the meantime.